http/fedora-27/ks-ppc64-openstack.cfg

```diff
@@ -0,0 +1,48 @@
+install
+cdrom
+lang en_US.UTF-8
+keyboard us
+network --bootproto=dhcp
+rootpw --plaintext fedora
+firewall --disabled
+authconfig --enableshadow --passalgo=sha512
+selinux --disabled
+timezone UTC
+bootloader --location=mbr
+text
+skipx
+zerombr
+clearpart --all --initlabel
+part prepboot --fstype=prepboot --asprimary --size=8 --ondisk=vda
+part / --fstype="ext4" --grow --size=100 --ondisk=vda
+bootloader --location=mbr --append="norhgb biosdevname=0"
+auth  --useshadow  --enablemd5
+firstboot --disabled
+reboot
+user --name=fedora --plaintext --password fedora
+url --url=http://ftp.osuosl.org/pub/fedora-secondary/releases/27/Server/ppc64/os/
+repo --name=updates --baseurl=http://ftp.osuosl.org/pub/fedora-secondary/updates/27/ppc64/
+
+%packages --ignoremissing
+bzip2
+man
+man-pages
+net-tools
+nfs-utils
+perl-libwww-perl
+rsync
+tar
+vim
+wget
+sudo
+%end
+
+%post
+dnf -y upgrade
+# update root certs
+wget https://raw.githubusercontent.com/bagder/curl/master/lib/mk-ca-bundle.pl
+perl mk-ca-bundle.pl /etc/pki/tls/certs/ca-bundle.crt
+rm certdata.txt mk-ca-bundle.pl
```

Comment:
Is this required for fedora 27? It was released just a month ago.

Also is this the best way to update the root certificate bundle?

Reply:
This can probably be removed.

Reply:
@doublej472 actually, I'd prefer to keep this in here so that we get the most updated root certs even if we build it later.